src/cta-button/style.scss

```diff
@@ -0,0 +1,55 @@
+/**
+ * #.# Styles
+ *
+ * CSS for both Frontend+Backend.
+ */
+
+@import "../scss/bulma-utilities";
+
+.cta-card {
+  &.card {
+    width: 100%;
+    max-width: 318px;
```

Comment:
ideally you should stick with using relative units and avoid hardcoding widths. try using rem here to match the rest of the pr